payments/base/arch/src/main/java/com/appcoins/payments/arch/error/UnknownErrorException.kt

```diff
@@ -0,0 +1,3 @@
+package com.appcoins.payments.arch.error
+
+class UnknownErrorException : Throwable()
```

Comment:
This exception is used for `else` case in repositories and not checked in the `BillingErrorMapper`.
`BillingErrorMapper` just checks for `ServerErrorException`, `ServerUnreachableException(java.net.UnknownHostException)` and `else` for everything else.
So looks like we don't need this class and can just re throw the original exception in repositories :-)